juniper_official/system/check-ntp-synchronization-status.rule

```diff
@@ -0,0 +1,185 @@
+            }                           
+            field stratum {
+                sensor ntp-status {
+                    path stratum;
+                }
+                type integer;
+                description "The stratum of the peer server. ";
+            }
+            /*
+             * Anomaly detection logic.
+             */
+            trigger ntp-sync {
+                frequency 1offset;
+                /*
+                 * Sets color to red when reference-id is INIT
+                 */
+                term is-sync-unspec {
+                    when {
+                        matches-with "$reference-id" INIT {
+                            ignore-case;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "NTP is in sync-unspec";
+                        }
+                    }
+                }
+                /*
+                 * Sets color to yellow when reference-id is LOCAL(0)
+                 */
+                term is-sync-local {
+                    when {              
+                        matches-with "$reference-id" "LOCAL\(0\)" {
+                            ignore-case;
+                        }
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "NTP state is sync local";
+                        }
+                    }
+                }
+                /*
+                 * Sets color to green when reference-id by default
+                 */
+                term is-in-sync {
+                    then {
+                        status {
+                            color green;
+                            message "NTP State is in sync with $reference-id";
+                        }
+                    }
+                }
+            }
+            variable status-info {
+                value .*;
+                description "Default is .*sync_ntp.*, Can also specify variable as .*sync_alarm.* or .*sync_unspec.*";
+                type string;
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                category advanced;
+                supported-healthbot-version 4.2.0;
+                supported-devices {
+                    juniper {
+                        operating-system junosEvolved {
+                            products ACX {
+                                platforms All {
+                                    releases 23.2R1 {
+                                        release-support min-supported-release;
+                                    }
+                                }
+                            }
+                            products PTX {
+                                platforms vardbeg {
```

Comment:
is platform vardbeg? I thought this name does not go anywhere in the show version or show chassis